std_msgs/Bool align_base_station
---
std_msgs/Bool success
